ntrans.py

```diff
@@ -33,28 +34,28 @@ def create_csv_file(
     return print("N-Trans CSV Glossary has been successfully saved to " + full_path)
 
 
-def machine_translate_ngrams(
-    ngrams: Dict[int, List[str]], user_choices: Dict[str, Any]
-) -> None:
+def machine_translate_ngrams(ngrams: Dict[int, List[str]], user_choices: Dict[str, Any], progress_queue: queue.Queue[float]) -> None:
```

Comment:
Python 3.7 doesn't understand `queue.Queue[float]`:

```
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import queue
>>> queue.Queue[float]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
```

So if you want to support Pythons that aren't the very latest but still not too old, add `from __future__ import annotations` to this file.